lms/resources/_js_config/file_picker_config.py

```diff
@@ -71,6 +71,19 @@ def google_files_config(cls, context, request, application_instance):
             "origin": origin,
         }
 
+    @classmethod
+    def microsoft_onedrive(cls, _context, request, application_instance):
+        enabled = application_instance.settings.get(
+            "microsoft_onedrive", "files_enabled"
+        )
+        if not enabled:
+            return {"enabled": False}
```

Comment:
I think this might be a good pattern to move to for all of these blobs of data. Although I'm not sure if it's safe as the front-end might now rely on all the other fields being present.

In this case the front-end doesn't exist yet, so we can get ahead of the curve.